src/targets/gpu/jit/reduce.cpp

```diff
@@ -169,13 +170,20 @@ struct simple_reduce_compiler : compiler<simple_reduce_compiler>
         options.kernel_name  = "reduce_kernel";
         std::string identity = "[](auto x) { return x; }";
         auto src             = interpolate_string(simple_reduce_kernel,
-                                      {{"reduction", v.at("reduction").to<std::string>()},
-                                       {"init", v.get("init", std::string{"0"})},
-                                       {"read", v.get("read", identity)},
-                                       {"write", v.get("write", identity)},
-                                       {"algo", algo},
-                                       {"transformers", make_transformer_args(vec)},
-                                       {"preamble", v.get("preamble", std::string{})}});
+                                                  {{"reduction", v.at("reduction").to<std::string>()},
+                                                   {"init", v.get("init", std::string{"0"})},
+                                                   {"read", v.get("read", identity)},
+                                                   {"write", v.get("write", identity)},
+                                                   {"algo", algo},
+                                                   {"transformers", make_transformer_args(vec)},
+                                                   {"preamble", v.get("preamble", std::string{})}});
+        // disable DPP for FP8 for now,, TODO: need to disable for Any FP8 types
+        if(std::any_of(inputs.begin(), inputs.end(), [](const auto& s) {
+               return s.type() == migraphx::shape::fp8e4m3fnuz_type;
+           }))
+        {
+            options.params += "-DMIGRAPHX_HAS_DPP=0 ";
+        }
```

Comment:
Is this needed? I would have thought the dpp reductions would've worked.

Reply:
Removed. It works.